siege/README.md

```diff
@@ -0,0 +1,43 @@
+# Envoy simple performance scripts.
+
+##Development-focused benchmarks:
+
+When updating code in Envoy it is often useful to try to reason about the
+performance impact of a change you are making. It is handy to have a suite of
+scripts to help measure performance. These scripts should not be used for
+absolute benchmarking.
+
+When running performance benchmarks, some basic principles can help give the
+best results:
+
+1. Run performance benchmarks on optimized builds: `bazel build -c opt source/exe:envoy-static`.
+2. Don't let any other software run on the machine at the same time.
+3. Make sure the configuration you are testing actually hits the code you are editing.
+4. If you are testing code changes, interleave runs between an envoy-static built from master
+   and one built from your branch.
+
+##About Siege:
+
+Siege is a github project that efficiently pummels an HTTP server with requests and measures
```

Comment:
Is Siege HTTTP/1 only?  I notice it's 1.1 in your config below and if so I think it's worth call out specifically here.